Changelogs for Digital Twin Manager Microservice 1024.1.1 (#4304)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1024-1-1-fixed-an-issue-where-the-title-and-description-of-provided-asset-definitions-were-being-reset-after-microservice-restart.md

---
date: ""

> title: "Fixed an issue where the title and description of provided Asset Definitions were being reset after microservice restart"

Reviewer: ```suggestion
title: "Title and description of pssset definitions no longer reset after microservice restart"
```

> title: "Fixed an issue where the title and description of provided Asset Definitions were being reset after microservice restart"

Reviewer 2: ```suggestion
title: "Title and description of asset definitions no longer reset after microservice restart"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-VSkj2iV9m"
      label: "Fix"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-2673"
version: "1024.1.1"
---

> Fixed an issue where title and description of provided Asset Definitions

Reviewer: ```suggestion
Previously, the title and description of provided asset definitions
```

> were incorrectly overwritten instead of being preserved from the

Reviewer: ```suggestion
were incorrectly reset after a microservice restart instead of being preserved from the
```

> tenant’s existing Asset Definition, while schema and other metadata are

Reviewer: ```suggestion
tenant’s existing asset definition, while schema and other metadata were
```

> updated from the library.

Reviewer: ```suggestion
updated from the library. This issue has been fixed. The title and description of asset definitions are no longer overwritten after a microservice restart.
```
